beluga/include/beluga/algorithm/raycasting/bresenham.hpp

```diff
@@ -215,27 +215,27 @@ class Bresenham2i {
     };
 
     /// Constructs point line.
-    Line() noexcept = default;
+    Line() = default;
 
     /// Default copy constructor.
-    Line(const Line&) noexcept = default;
+    Line(const Line&) = default;
 
     /// Default move constructor.
-    Line(Line&&) noexcept = default;
+    Line(Line&&) = default;  // NOLINT(performance-noexcept-move-constructor)
 
     /// Default copy assignment operator overload.
-    Line& operator=(const Line&) noexcept = default;
+    Line& operator=(const Line&) = default;
 
     /// Default move assignment operator overload.
-    Line& operator=(Line&&) noexcept = default;
+    Line& operator=(Line&&) = default;  // NOLINT(performance-noexcept-move-constructor)
```

Comment:
same, use rule of zero here

Reply:
Done in https://github.com/Ekumen-OS/beluga/pull/209/commits/19efb3221b06f5fddc7ea3fcbc1cc3ef60f97bc5.